feature/split/src/main/java/com/blitzsplit/split/presentation/viewmodel/SplitEvent.kt

```diff
@@ -0,0 +1,6 @@
+package com.blitzsplit.split.presentation.viewmodel
+
+sealed class SplitEvent {
+    data class ChangeBillTitleTextFieldValue(val value: String): SplitEvent()
+    data class ChangeBillValueTextFieldValue(val value: String): SplitEvent()
+}
```

Comment:
https://www.youtube.com/shorts/GzKzkMGFZAs?feature=share

```suggestion
sealed interface SplitEvent {
    data class ChangeBillTitleTextFieldValue(val value: String): SplitEvent
    data class ChangeBillValueTextFieldValue(val value: String): SplitEvent
}
```